openedx/core/lib/mobile_utils.py

```diff
@@ -29,7 +29,7 @@ def is_request_from_mobile_app(request):
         user_agent = request.META.get('HTTP_USER_AGENT')
         if user_agent:
             for user_agent_regex in settings.MOBILE_APP_USER_AGENT_REGEXES:
-                if re.match(user_agent_regex, user_agent):
+                if re.search(user_agent_regex, user_agent):
```

Comment:
`re.match` checks for a match only at the beginning of the string while `re.search` checks for a match anywhere in the string and mobile user agent does not always have matching string at the beginning. I wonder why it had been like this as it always returns `False` even if request is from mobile.